src/alpha-lab/multiple-rim.md

```diff
@@ -4,93 +4,166 @@ permalink: /alpha-lab/multiple-rim/
+  <div style="flex: 50%; padding: 5px;">
+    <div style="position: relative; padding-bottom: 75%; height: 0; margin-bottom: -35%;">
+      <iframe style="position: absolute; width: 50%; height: 50%; left: 50%; transform: translateX(-50%); border: none;" src="https://www.youtube.com/embed/fmy9F8Q9eW0?si=F7q399iZHGW2kArv"></iframe>
+    </div>
+  </div>
 </div>
 
-During the recording, the user looked at the same furniture and parts of the room multiple times. We suggest you focus on
-specific [sections](/enrichments/#enrichment-sections) of the recording based on which part of the
-room the user is exploring.
+<div style="margin-bottom: 5px;"></div>
 
-For this recording, we used the following [event annotations](/invisible/basic-concepts/events) to run five Reference Image Mapper enrichments:
+3. **Eye Tracking Recordings:** Make an eye tracking recording while the participant(s) freely explore and visually interact with various elements within the environment. (You can of course make these prior to the reference images and scanning recordings).
 
-- Cupboard: `cupboard.begin` and `cupboard.end`
-- Desk: `desk.begin` and `desk.end`
-- Kitchen: `kitchen.begin` and `kitchen.end`
-- TV: `tv.begin` and `tv.end`
-- Table: `table.begin` and `table.end`
+<div style="margin-bottom: 5px;"></div>
 
-## Final results
+4. **Add Custom Events:** During the eye tracking recording, users may focus on specific furniture or parts of the room multiple times. By adding custom [event](/neon/basic-concepts/events) annotations corresponding to these areas or objects, you can create [sections](/enrichments/#enrichment-sections) for the enrichments to be performed. This approach allows you to run each enrichment only on the portion of the recording where a certain object is present. For this guide, we used the following event annotations to run five Reference Image Mapper enrichments:
+    - Desk: `desk.begin` and `desk.end`
+    - TV area 1: `tv1.begin` and `tv1.end`
+    - TV area 2: `tv2.begin` and `tv2.end`
+    - Table: `table.begin` and `table.end`
+    - Kitchen: `kitchen.begin` and `kitchen.end`
+    - Cupboard: `cupboard.begin` and `cupboard.end`
 
-It may take several minutes to run these enrichments depending on how long your recordings are. Once everything is finished, you can visualize how gaze is simultaneously mapped both on the recording and the reference images from the Project Editor view (as shown in the video at the very beginning of this guide).
+5. **Create and run the enrichments:** You will need to create a separate enrichment for each reference image. A reasonable naming scheme *could* correspond to each area of the environment, like ‘cupboard’, ‘desk’ etc. In the temporal selection of each enrichment, be sure to use the appropriate events labels. E.g. for ‘cupboard’, you would use `cupboard.begin` and `cupboard.end`. Now, run the enrichments to map the subject's gaze from the recording onto the multiple reference images you captured.
```

Comment:
Is the note about naming scheme necessary?

Reply:
This is suitable for our customers' level of understanding imo